io.openems.edge.pvinverter.kaco.blueplanet/src/io/openems/edge/pvinverter/kaco/blueplanet/Config.java

```diff
@@ -19,6 +19,9 @@
 	@AttributeDefinition(name = "Modbus-ID", description = "ID of Modbus bridge.")
 	String modbus_id() default "modbus0";
 
+	@AttributeDefinition(name = "Modbus Unit-ID", description = "The Unit-ID of the Modbus device.")
+	int modbusUnitId() default 3;
```

Comment:
We should set a default value of 1 to stay backwards compatible with Modbus/TCP. Maybe add a hint here or in the readme that it has to be set explicitely for Modbus/RTU connections.

Reply:
Just to be clear, the NX3 has a Modbus ID of 3 for both TCP and RTU. On TCP, it responds to Modbus calls on ID 3 *and* ID 1.  
Is the Modbus ID default value 1 needed for other Kaco devices? I only tested with the NX3, so I have no knowledge on how other Kaco devices operate.

Reply:
I did not test with different Unit-IDs than 1. Anyhow it would (could?) be a breaking change if we now just change it to 3, which sounds dangerous. 

From my point of view:
Either we should get a note from KACO directly whether '3' is always the default for their SunSpec Modbus devices and '1' just happens to work. Or we should document it in the Config.java and/or in the readme, that typically this parameter has to be set to '3'.

Reply:
I agree with your point of view.  
I will go with setting the default to 1, and documenting it in the config & readme ("If 1 does not work, try 3").